newshomepages/sources/sites.csv

```diff
@@ -588,7 +594,7 @@ YCityNewsOnline,https://ycitynews.com/,Y-City News,Zanesville,America/New_York,U
 YahooFinance,https://finance.yahoo.com/,Yahoo Finance,New York City,America/New_York,US,en,,
 YahooNews,https://news.yahoo.com/,Yahoo! News,New York City,America/New_York,US,en,us-national,
 YediotAhronot,https://www.yediot.co.il/,Yedioth Ahronoth,Tel Aviv,Asia/Tel_Aviv,IL,he,,
-fiquemsabendo,https://fiquemsabendo.substack.com/,Fiquem Sabendo,Sao Paulo,America/Sao_Paulo,BR,pt,,
+_fiquemsabendo,https://fiquemsabendo.substack.com/,Fiquem Sabendo,Sao Paulo,America/Sao_Paulo,BR,pt,,

```

Comment:
We cut the leading `_` on this solve an archive.org bug. Can you catch up with the tweak in main?